exp/views/responses.py

```diff
@@ -1200,27 +1200,32 @@ def render_to_response(self, context, **response_kwargs):
                     # psych-DS files use "keyword" formatting
                     keywords = {
                         "study": study_name_for_files(study.name),
-                        "response": resp.id,
-                        "child": resp.child_id,
+                        "response": str(resp.uuid).replace("-",""),
+                        "child": str(resp.child.uuid).replace("-",""),
```

Comment:
This is exactly what I would've done. But after reading about UUIDs a bit, I think you might be able to do something like `resp.uuid.hex`? https://docs.python.org/3/library/uuid.html#uuid.UUID.hex